doc/api/deprecations.md

```diff
@@ -2497,6 +2497,19 @@ Type: Runtime
 Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 `Promise` instead, or a listener to the worker’s `'exit'` event.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http finished
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28679
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+`finished` is what `ended` is in streams spec.
```

Comment:
The recommendation is unclear. It should tell users what to do to avoid usage of the deprecated functionality.